Gui/SequenceFileDialog.cpp

```diff
@@ -465,19 +465,21 @@ SequenceFileDialog::SequenceFileDialog( QWidget* parent, // necessary to transmi
     _selectionLineEdit = new FileDialogLineEdit(_selectionWidget);
     _selectionLayout->addWidget(_selectionLineEdit);
 
-    if ( (mode == SequenceFileDialog::eFileDialogModeOpen) || (mode == SequenceFileDialog::eFileDialogModeDir) ) {
-        _openButton = new Button(tr("Open"), _selectionWidget);
-    } else {
-        _openButton = new Button(tr("Save"), _selectionWidget);
+    if (_dialogMode == eFileDialogModeSave) {
+        _fileExtensionCombo = new ComboBox(_selectionWidget);
+        for (int i = 0; i < _filters.size(); ++i) {
+            _fileExtensionCombo->addItem( _filters[i] );
+        }
+        _selectionLayout->addWidget(_fileExtensionCombo);
+        QObject::connect( _fileExtensionCombo, SIGNAL(currentIndexChanged(int)), this, SLOT(onFileExtensionComboChanged(int)) );
+        if (isSequenceDialog) {
+            int idx = _fileExtensionCombo->itemIndex( QString::fromUtf8("jpg") );
```

Comment:
We should probably not set the output image format to jpg by default.
Default should probably be EXR, TIFF or DPX for a compositor